v0/destinations/marketo/transform.js

```diff
@@ -611,6 +619,7 @@ const processRouterDest = async inputs => {
   const respList = await Promise.all(
     inputs.map(async input => {
       try {
+        input.metadata.destInfo = { authKey: input.destination.ID };
```

Comment:
This would be messy, as @utsabc noted [here](https://github.com/rudderlabs/rudder-transformer/pull/1488#discussion_r1007847457).
We should probably wait for this PR to get merged.

Reply:
@ujjwal-ab can you address this?

Reply:
Ya, it's addressed.